data-otservbr-global/scripts/actions/other/others/quest_system1.lua

```diff
@@ -34,7 +34,7 @@ local function copyContainerItem(originalContainer, newContainer)
 		local originalItem = originalContainer:getItem(i)
 		local newItem = Game.createItem(originalItem.itemid, originalItem.type)
 		if not newItem then
-			Spdlog.error("[questSystem1.copyContainerItem] failed to create item " .. originalItem.itemid)
+			logger.error("[questSystem1.copyContainerItem] failed to create item " .. originalItem.itemid)
```

Comment:
Você pode modificar também a estrutura do log, já que o logger suporta a sintaxe do cpp:
```suggestion
			logger.error("[questSystem1.copyContainerItem] failed to create item: {}", originalItem.itemid)
```